pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/ScalarTransformFunctionWrapper.java

```diff
@@ -444,4 +446,10 @@ private void getNonLiteralValues(ValueBlock valueBlock) {
       }
     }
   }
+
+  @Nullable
```

Comment:
We don't need to define this function because we can inherit the function from the subclass.